### PR TITLE
Fix supporting eu-north-1 region

### PR DESCRIPTION
**Pull request by kvasukib**
*Issue #, if available:*

#797 

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-android-sdk-core/src/main/java/com/amazonaws/regions/RegionDefaults.java [hunk 1]
```diff
@@ -422,6 +422,31 @@
...
+        updateRegion(region, "kinesis", "kinesis.eu-north-1.amazonaws.com", false, true);
+        updateRegion(region, "kms", "kms.eu-north-1.amazonaws.com", false, true);
+        updateRegion(region, "lambda", "lambda.eu-north-1.amazonaws.com", false, true);
+        updateRegion(region, "logs", "logs.eu-north-1.amazonaws.com", false, true);
+        updateRegion(region, "polly", "polly.eu-north-1.amazonaws.com", false, true);
+        updateRegion(region, "s3", "s3.eu-north-1.amazonaws.com", false, true);
+        updateRegion(region, "sns", "sns.eu-north-1.amazonaws.com", false, true);
+        updateRegion(region, "sqs", "sqs.eu-north-1.amazonaws.com", false, true);
+        updateRegion(region, "sts", "sts.amazonaws.com", false, true);
+
```
Comment (desokroshan) [inline aws-android-sdk-core/src/main/java/com/amazonaws/regions/RegionDefaults.java]: Can we remove the code duplication here?

File: aws-android-sdk-core/src/main/java/com/amazonaws/regions/RegionDefaults.java [hunk 2]
```diff
@@ -422,6 +422,31 @@
         updateRegion(region, "sns", "sns.us-gov-west-1.amazonaws.com", false, true);
         updateRegion(region, "sqs", "sqs.us-gov-west-1.amazonaws.com", false, true);
         updateRegion(region, "sts", "sts.amazonaws.com", false, true);
+
```
Comment (desokroshan) [inline aws-android-sdk-core/src/main/java/com/amazonaws/regions/RegionDefaults.java]: Can we remove the duplication here?

File: aws-android-sdk-core/src/main/java/com/amazonaws/regions/RegionDefaults.java [hunk 3]
```diff
@@ -422,6 +422,31 @@
...
+        region = new Region("eu-north-1", "amazonaws.com");
+        ret.add(region);
+        updateRegion(region, "autoscaling", "autoscaling.eu-north-1.amazonaws.com", false, true);
```
Comment (minbi) [inline aws-android-sdk-core/src/main/java/com/amazonaws/regions/RegionDefaults.java]: autoscaling.eu-north-1.amazonaws.com
dynamodb.eu-north-1.amazonaws.com
ec2.eu-north-1.amazonaws.com
elasticloadbalancing.eu-north-1.amazonaws.com
firehose.eu-north-1.amazonaws.com
iot.eu-north-1.amazonaws.com
kinesis.eu-north-1.amazonaws.com
kms.eu-north-1.amazonaws.com
lambda.eu-north-1.amazonaws.com
logs.eu-north-1.amazonaws.com
s3.eu-north-1.amazonaws.com
sns.eu-north-1.amazonaws.com
sqs.eu-north-1.amazonaws.com
sts.eu-north-1.amazonaws.com

**Comment by kkflf** [comment]
Hi, any updates on this pull-request? I would love to be able to use S3 Stockholme on Android soon. 👍

**Pull request merged**